### DISCUSSION TITLE
Please add query parameter in the tags API endpoint to fetch tags matching a searched word.

### DISCUSSION THREAD

**Discussion by mayank12gt** [forem/forem#19550] [Feature Requests]
It would be great if there was a query parameter in the [tags](https://dev.to/api/tags) API endpoint to fetch only those tags which have the searched word in their title. The [tags](https://dev.to/api/tags) API simply returns all the available tags and currently there is no way to get only those tags which match a search query, for example "Java".
What I want is to be able to get only those tags which match my search query. Ex- If I pass a search parameter "Java" with the [tags](https://dev.to/api/tags) API, it should return all and only those tags which have Java in their title.

I am building an app that fetches articles from the dev.to API and it would be really helpful, if this feature was added.

**Comment by michael-tharrington** [comment]
Thanks so much for writing this request @mayank12gt. I'll make sure that the team sees this one!
